Rock_paper_scissors/main.js

```diff
@@ -0,0 +1,28 @@
+const arr = ['rock', 'paper', 'scissor'];
+let wins = 0;
+let losses = 0;
+let ties = 0;
+
+function ABC(value) {
+    var rand = arr[Math.floor(Math.random() * 3)];
+    document.getElementById("print_result").innerHTML = `You choose ${value}, computer choose ${rand}.<br/><br/>`;
+
+    if (value == 'rock' && rand == "scissor" ||
+        value == "paper" && rand == "rock" ||
+        value == "scissor" && rand == "paper") {
+        document.getElementById("win").innerHTML = wins += 1;
+        document.getElementById("print_result").innerHTML += `You Win`;
+
+    } else if (rand == 'rock' && value == "scissor" ||
+        rand == "paper" && value == "rock" ||
+        rand == "scissor" && value == "paper") {
+        document.getElementById("lose").innerHTML = losses += 1;
+        document.getElementById("print_result").innerHTML += `Computer Win`;
+
+    } else {
+        document.getElementById("tie").innerHTML = ties += 1;
+        document.getElementById("print_result").innerHTML += `it's a Tie`;
+
+    }
```

Comment:
Avoid using repetitive piece of code, store it in one variable and use it,  here 
update the code using ternary operator instead of if..else
what i can see is there are only three cases when user can win:
you_win = ( (you === "rock") && (opponent === "scissor") ||
(you === "paper") && (opponent === "rock") ||
(you === "scissor") && (opponent === "rock") ) // returns true for your win cases only.
.
.
so (you === opponent) ? "draw message" : you_win ? call a method for updating score and message for user : call method for updating score and message fro computer;